tuning/punet_autotune.py

```diff
@@ -66,12 +66,13 @@ def get_model_compile_command(
     ) -> list[str]:
         mlir_spec_path = candidate_tracker.spec_path
         assert mlir_spec_path is not None
+        script_dir = Path(__file__).resolve().parent
         target_dir = mlir_spec_path.resolve().parent.parent.parent
         output_name = f"unet_candidate_{candidate_tracker.candidate_id}.vmfb"
         command = [
             "timeout",
             "300s",
-            "../int8-model/compile-punet-base.sh",
+            (script_dir / "../int8-model/...").as_posix(),
```

Comment:
This should use the .sh script, I only used `...` because I was to lazy to type the full name 🤦‍♂️